qiskit_nature/second_q/operators/fermionic_op.py

```diff
@@ -148,6 +148,11 @@ class FermionicOp(SparseLabelOp):
             considered a lower bound, which means that mathematical operations acting on two or more
             operators will result in a new operator with the maximum number of spin orbitals of any
             of the involved operators.
+    .. note::
+
+        A FermionicOp can contain Parameters. However, a FermionicOp containing Parameters
+        does not support the following methods:
+        - ``is_hermitian``
```

Comment:
```suggestion
        - ``is_hermitian``
        - ``to_matrix``
```
This method also does not work as long as we do not refactor the `to_matrix` implementation to use the `sparse` library instead of `scipy.sparse`

Reply:
Done.